homepage/view.php

```diff
@@ -18,28 +18,28 @@
 		<section class="column-left">
 			<div class="box-panel">
 				<div class="panel-cover">
-					<img src="/static/logo/ribbon-long.svg" alt="wuwana ribbon">
+					<img src="/static/logo/ribbon-long.svg" alt="wuwana logo ribbon">
 				</div>
 				<section>
 					<p><?php echo TEXT[9] ?></p>
 					<a class="button-main center" target="_blank"
-					 href="https://medium.com/wuwana/qué-es-wuwana-7c2defac2302">
+					 href="https://medium.com/wuwana/qué-es-wuwana-7c2defac2302" rel="noopener">
 						<?php echo TEXT[10] ?>
 					</a>
 				</section>
 				<hr>
 				<section class="contact-section">
-					<h3><?php echo TEXT[11] ?></h3>
+					<label><?php echo TEXT[11] ?></label>
```

Comment:
Why a label?
Label element represents a caption for an item in a user interface and are associated with <input>